File-Formats/DocIO/Working-with-Find-and-Replace.md

```diff
@@ -799,6 +799,116 @@ Xamarin.Forms.DependencyService.Get<ISave>().SaveAndView("Replace.docx", "applic
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Find-and-Replace/Find-and-replace-paragraphs-with-text).
 
+## Find next and replace with formatted text
+
+The following code example illustrates how to replace the text extended to several paragraphs with particular text in the document:
+
+{% tabs %}  
+
+{% highlight c# tabtitle="C#" %}
+//Load an existing Word document. 
+using (WordDocument document = new WordDocument("Input.docx", FormatType.Docx))
+{
+    //Access the specific paragraph in the Word document.
+    TextBodyItem textBodyItem = document.Sections[0].Paragraphs[3] as WParagraph;
+    //Find the next occurrence of specified text from the previous paragraph.
+    TextSelection textSelections = document.FindNext(textBodyItem, new Regex("Adventure Works Cycles"));
+    //Replace the text extended to two paragraphs with the above selected text.
+    document.ReplaceSingleLine("CompanyName", textSelections, true, true);
+    //Save the Word document.
+    document.Save("Sample.docx");
+}           
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET" %}
+'Load an existing Word document. 
+Using document As WordDocument = New WordDocument("Input.docx", FormatType.Docx)
+    'Access the specific paragraph in the Word document.
```

Comment:
'Access the specific paragraph in a Word document.

Reply:
@Chrispineagunja -Done